content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,17 @@ menu:
     parent: About the project
 ---
 
+## v1.8.8 [2021-08-03]
+The InfluxDB Enterprise 1.8.8 release builds on the InfluxDB OSS 1.8.8 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v188-2021-08-0?).
+
+### Bug fixes 
+
+- Prevent silently dropped writes with overlapping shards.
+- Restore portable backup bug.
+- Systemd-startup script should be executable by group and others.
+
```

Comment:
@mluu12, if lines 19-21 are the InfluxDB OSS 1.8.8 fixes, then we can delete them here